python/django-4/README.md

```diff
@@ -0,0 +1,40 @@
+# Python OpenTelemetry
+
+## Installation
+
+```
+python -m pip install -r requirements.txt
+```
+
+Run dependency services using Docker:
+
+```
+docker-compose up
+```
+
+## Setup
+
+```
+python manage.py makemigrations
+python manage.py migrate
+```
```

Comment:
Is this README still relevant? It seems to me that these instructions are no longer necessary (`rake app=python/django-4 app:up` should do all this, via `commands/run` and `processmon`)